pagai/services/database_explorer.py

```diff
@@ -192,27 +197,31 @@ def get_owners(self):
             result = connection.execute(sql_query).fetchall()
         return [r["owners"] for r in result]
 
-    def get_db_schema(self):
+    def get_db_schema(self, owner: str):
```

Comment:
You also need to change the route `get_db_schema` in `views.py`

Reply:
Also, what do you think about changing all the `get_db_schema` to `get_owner_schema`?

Reply:
thanks! done